microsoft/PowerToys
# Glyphs for plugin settings

## Summary of the Pull Request

**What is this about:**
![image](https://user-images.githubusercontent.com/17161067/108527327-d2ccb480-72da-11eb-90ad-5798322ba3dd.png)

**What is include in the PR:** 
- Add glyph filed to `plugin.json` file
- Delete dark and light icons from plugin settings

**How does someone test / validate:** 
- Delete `PowerToys Run\settings.json`
- Verify that glyphs are rendered in the settings app

@niels9001 , glyphs are not properly aligned, especially the calculator. Any idea what is happening? 
I have not found the Windows Walker glyph so for now, we use a different one. If we do not find the same glyph, then I guess we can replace it with the appropriate one everywhere.

## Quality Checklist

- [X] **Linked issue:** #9797

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
